plugins/woocommerce/tests/e2e-pw/bin/woocommerce-cleanup.php

```diff
@@ -341,14 +337,9 @@ function () {
 			'wc-cleanup/v1',
 			'/reset',
 			array(
-				'methods'             => 'POST',
+				'methods'             => 'GET',
 				'callback'            => 'wc_cleanup_reset_site_via_api',
 				'permission_callback' => function () {
-					// Verify the nonce before processing the request.
-					if ( ! isset( $_POST['wc_cleanup_reset_nonce'] ) || ! wp_verify_nonce( sanitize_text_field( wp_unslash( $_POST['wc_cleanup_reset_nonce'] ) ), 'wc_cleanup_reset_action' ) ) {
-						return new WP_Error( 'rest_forbidden', esc_html__( 'Nonce verification failed', 'woocommerce' ), array( 'status' => 403 ) );
-					}
-
 					$provided_key = isset( $_GET['key'] ) ? sanitize_text_field( wp_unslash( $_GET['key'] ) ) : '';
```

Comment:
I changed this to a GET from a POST to simplify authentication.  With a POST, you need to employ a nonce check... and we're not really POSTing anything to the server, we're just triggering the plugin.